en/asgardeo/docs/guides/api-security/aws/securing-aws-http-apis.md

```diff
@@ -0,0 +1,279 @@
+ROLE_ARN=$(aws iam get-role --role-name LambdaExecutionRole --query 'Role.Arn' --output text)
+echo "Created Lambda IAM Role ARN: $ROLE_ARN"
+```
+
+### 2. Write the Lambda Function Code
+```bash
+cat > index.js << 'EOF'
+exports.handler = async (event) => {
+    const claims = event?.requestContext?.authorizer?.jwt?.claims || {};
+    const userName = claims.name || claims.preferred_username || claims.email || "User";
+    return {
+        statusCode: 200,
+        body: JSON.stringify({ message: `Welcome ${userName}!` })
+    };
+};
+EOF
+
+zip function.zip index.js
+```
+
+### 3. Deploy the Lambda Function
+```bash
+aws lambda create-function \
+  --function-name PharmacyUserFunction \
+  --runtime nodejs18.x \
+  --handler index.handler \
+  --zip-file fileb://function.zip \
+  --role $ROLE_ARN
+
+echo "Lambda function 'PharmacyUserFunction' deployed."
+```
+
+## Step 2: Configure API Gateway Integration
+
+### 1. Create HTTP API
+```bash
+API_ID=$(aws apigatewayv2 create-api \
+  --name "PharmacyUserAPI" \
+  --protocol-type HTTP \
+  --query 'ApiId' --output text)
+echo "Created API Gateway with ID: $API_ID"
+```
+
+### 2. Integrate Lambda with API
+```bash
+FUNCTION_ARN=$(aws lambda get-function --function-name PharmacyUserFunction --query 'Configuration.FunctionArn' --output text)
+
+INTEGRATION_ID=$(aws apigatewayv2 create-integration \
+  --api-id $API_ID \
+  --integration-type AWS_PROXY \
+  --integration-method POST \
+  --integration-uri $FUNCTION_ARN \
+  --payload-format-version 2.0 \
+  --query 'IntegrationId' --output text)
+echo "Created API integration with ID: $INTEGRATION_ID"
+```
+
+### 3. Create API Route
+```bash
+ROUTE_ID=$(aws apigatewayv2 create-route \
+  --api-id $API_ID \
+  --route-key "GET /user" \
+  --target "integrations/$INTEGRATION_ID" \
+  --authorization-type NONE \
+  --query 'RouteId' --output text)
+echo "Created route GET /user with ID: $ROUTE_ID"
+```
+
+### 4. Grant Permission to API Gateway
+```bash
+REGION=$(aws configure get region)
+ACCOUNT_ID=$(aws sts get-caller-identity --query Account --output text)
+
+aws lambda add-permission \
+  --function-name PharmacyUserFunction \
+  --statement-id AllowInvokeByApiGateway \
+  --action lambda:InvokeFunction \
+  --principal apigateway.amazonaws.com \
+  --source-arn "arn:aws:execute-api:$REGION:$ACCOUNT_ID:$API_ID/*/GET/user"
+```
+
+### 5. Deploy the API
+```bash
+aws apigatewayv2 create-stage --api-id $API_ID --stage-name dev --auto-deploy
+echo "API deployed to stage 'dev'"
+```
+
+## Step 3: Obtain a Bearer Token for Asgardeo Manage App
+
+You will need a Client ID and Client Secret from an Asgardeo Manage application that can be used to call Asgardeo’s management APIs. You can use an existing application or create a new Manage application in the Asgardeo console with the Client Credentials grant enabled. Make sure the application has the scope internal_application_mgt_create (this scope is required to create a new application via the API).
```

Comment:
```suggestion
You will need a Client ID and Client Secret from an Asgardeo application that is authorized to invoke Asgardeo management APIs. You can use an existing application or create a new application in the Asgardeo console with the Client Credentials grant enabled. Make sure the application has the scope `internal_application_mgt_create` (this scope is required to create a new application via the API).
```

Reply:
Addressed in [e4fc8aa](https://github.com/wso2/docs-is/pull/5253/commits/e4fc8aa8a7f3140b9f3447455ff995b2ad2ef2ac)